src/libs/antares/study/parts/short-term-storage/container.h

```diff
@@ -35,11 +35,14 @@ namespace Antares::Data::ShortTermStorage
 class STStorageInput
```

Comment:
The name container is used for other types of productions in src/libs/antares/study/parts/

Changing the name would be appropriate but would break consistency 
@flomnes what's the best option ?